README.md

```diff
@@ -668,6 +668,35 @@ export default {
 
 ### Angular / Ionic examples coming soon
 
+### Integration with Apollo Client (No offline support)
+
+In order to use AppSync with Apollo Client you need to install `aws-appsync-auth-link` and `aws-apsync-subscription-link` (if you want real time subscriptions)
```

Comment:
Let's clarify this - something like "For versions of the Apollo client newer than X.Y.Z you can use custom links for Authorization and Subscriptions. Offline support is not available for these newer versions."